dockerspawner/dockerspawner.py

```diff
@@ -147,6 +147,18 @@ def client(self):
         )
     )
 
+    internal_ip_network_name = Unicode(
+        "bridge",
+        config=True,
+        help=dedent(
+            """
+            Retrieve the internal IP from this docker network. Defaults to the default docker
```

Comment:
It is the name. For example created with _docker network create <network-name>_. I did put the internal_ip in front of it as it only makes sense to use this setting if you use _use_internal_ip=True_. Would you prefer something like just _network_name_ or _internal_network_name_?